sdk/storage/azure-storage-queue/src/main/java/com/azure/storage/queue/QueueServiceAsyncClient.java

```diff
@@ -250,10 +278,33 @@ public QueueAsyncClient getQueueAsyncClient(String queueName) {
      * <p>For more information, see the
      * <a href="https://docs.microsoft.com/en-us/rest/api/storageservices/get-queue-service-properties">Azure Docs</a>.</p>
      *
-     * @return Storage account Queue service properties
+     * @return Storage account {@link StorageServiceProperties Queue service properties}
```

Comment:
>Queue service properties [](start = 63, length = 24)

This should probably be outside the parenthesis

Reply:
For JavaDoc `@link` after the class or method reference is an alias, when generated it will show up as:

`Storage account Queue service properties`

where Queue service properties is a hyperlink to `StorageServiceProperties`

Reply:
Ok, I see that's fine then

---
In reply to: [314902103](https://github.com/Azure/azure-sdk-for-java/pull/4934#discussion_r314902103) [](ancestors = 314902103)